nix/nixos/cardano-cluster-service.nix

```diff
@@ -5,173 +5,277 @@
 
 with lib; with builtins;
 let
-  cfg = config.services.cardano-cluster;
+  ### Service configs
+  ccfg = config.services.cardano-cluster;
   ncfg = config.services.cardano-node;
-  node-ids = range 0 (cfg.node-count - 1);
-  cardano-node = ncfg.package;
+  pcfg = config.services.byron-proxy;
```

Comment:
this cause
```
The option `services.byron-proxy' defined in `/nix/store/zls9vlzkcz0ny6h05frm699i52k1nldp-source/nix/nixos/cardano-cluster-service.nix' does not exist.
```
when importing `cardano-node + "/nix/nixos"` on a node without byron-proxy module imported.

Reply:
hmm, but `cardano-cluster-service.nix` shouldn't be used by anything but the CI!

Reply:
Fixed in https://github.com/input-output-hk/cardano-node/pull/351 & tip of https://github.com/input-output-hk/cardano-ops/pull/23